partner_event/tests/test_event_registration.py

```diff
@@ -73,3 +75,14 @@ def test_data_update(self):
         self.partner_01.write({'email': 'new@test.com'})
         self.assertEqual(
             event_2.registration_ids.email, 'new@test.com')
+
+    def test_delete_registered_partner(self):
+        # We can't delete a partner with registrations
+        with self.assertRaises(IntegrityError), self.cr.savepoint():
+            self.partner_01.unlink()
+        # Create a brand new partner and delete it
```

Comment:
I think it makes more sense to delete `partner_01` registrations and then try to delete it again, for the test purposes.